### DISCUSSION TITLE
How do I change all ocurrences of a word at once?

### DISCUSSION THREAD

**Discussion by thiskappaisgrey** [meow-edit/meow#366] [Q&A] [answered]
For example if I have:
```
var a = ....

if ( a == 1) {
....
}
a += ...
```

And I want to change the variable name of a very quickly? If I press "W" with cursor on `a`, all the other occurrences of `a` is now highlighted. Now, how do I then change the var name `a` to something else? 

Right now, I just use `lsp-rename` or `meow-grab` the whole buffer then press W on the word to change it or use search-and-replace, but those ways are slow. Just want to know if there's a faster way. Thanks.

**Comment by DogLooksGood** [comment, marked as answer]
No, there's no faster way in Meow, you have to roll your own. A scope is necessary for batch editing, so you have to grab first. If it's just 3 places, you can do: `mark`, `change`, enter a new variable name, `esc`, `save`, `search`, `replace` `search`, `replace`.

**Comment by fbergroth** [comment]
#375 would make this easier, since you can grab without loosing the marked symbol.
